ui/app/src/utils/path.ts

```diff
@@ -105,6 +105,14 @@ export function getParentsFromPath(path: string, rootPath: string): string[] {
   return [rootPath, ...splitPath.map((value, i) => `${rootPath}/${splitPath.slice(1, i + 1).join('/')}`).splice(2)];
 }
 
+export function getIndividualPaths(path: string, rootPath: string) {
```

Comment:
why do you need the rootpath as a separate argument?

Reply:
I was using two different rootpath. Now only one, we can remove it, if you want to